src/components/Fields/Disclaimer/Disclaimer.tsx

```diff
@@ -1,31 +1,40 @@
+import { useParams } from "react-router";
+import { useDisclaimers } from "src/api/Disclaimer/useDisclaimers";
 import { PCOL } from "src/api/PCOL/types";
 import BACCheckbox from "src/components/BaseFormFields/BACCheckbox";
 
+type DisclaimerOptions = {
+  id: string;
+  text: string;
+  info?: string;
+};
+
 export const Disclaimer = () => {
-  // const Disclaimers = useDisclaimers();
-  const Disclaimers = [
-    {
-      id: "RFP Disclaimer",
-      text: "RFP Disclaimer",
-      info: "Something about RFP Disclaimers",
-    },
-    {
-      id: "Contract Price Disclaimer",
-      text: "Contract Price Disclaimer",
-      info: "Long text describing this Disclaimer",
-    },
-    {
-      id: "Contract Price FAR 32.2 Disclaimer",
-      text: "Contract Price FAR 32.2 Disclaimer",
-      info: "A description about this FAR 32.2 Disclaimer",
-    },
-  ];
+  const { program } = useParams();
+  const GlobalDisclaimers = useDisclaimers();
+  const ProgramDisclaimers = useDisclaimers(program);
+
+  const Disclaimers: DisclaimerOptions[] = [];
+  GlobalDisclaimers.data?.forEach((item) =>
+    Disclaimers.push({ id: item.Title, text: item.Title, info: item.Statement })
+  );
+  ProgramDisclaimers.data?.forEach((item) =>
+    Disclaimers.push({ id: item.Title, text: item.Title, info: item.Statement })
+  );
 
   return (
-    <BACCheckbox<PCOL>
-      name="Disclaimer"
-      labelText="Disclaimer(s)"
-      options={Disclaimers}
-    />
+    <>
+      <BACCheckbox<PCOL>
+        name="Disclaimer"
+        labelText="Disclaimer(s)"
+        options={Disclaimers}
+      />
+      {(GlobalDisclaimers.isError || ProgramDisclaimers.isError) && (
+        <>
+          There was an error fetching Disclaimer statements. Refresh the page to
+          try again, or contact support.
+        </>
+      )}
+    </>
```

Comment:
Consider adding a wrapper for the data loading -- and then don't show any Disclaimers if either case fails. Right now if one fails but the other does not, this message just appears at the bottom of the list of either global or local depending on which failed.
Hopefully they don't have too many program specific ones, as the list of check boxes is decently long with just the global ones. Should we consider adding a way to delineate between which ones were Global, and which were Program Specific?

Reply:
I'm good with just adding in which level(s) failed to load. That way they are aware and can take action appropriately. Maybe make the text the error red color -- so that it also stands out a bit more.